gst/nnstreamer/elements/gsttensor_split.c

```diff
@@ -594,25 +597,36 @@ gst_tensor_split_set_property (GObject * object, guint prop_id,
       guint i;
       const gchar *param = g_value_get_string (value);
       gchar **strv = g_strsplit_set (param, ",.;/", -1);
+      GArray *tensorseg = split->tensorseg;
+
       split->num_tensors = g_strv_length (strv);
-      split->tensorseg =
-          g_array_sized_new (FALSE, FALSE, sizeof (tensor_dim *),
-          split->num_tensors);
+      if (NULL == tensorseg) {
+        split->tensorseg =
+            g_array_sized_new (FALSE, FALSE, sizeof (tensor_dim *),
+            split->num_tensors);
+
+        for (i = 0; i < split->num_tensors; i++) {
+          tensor_dim *d = g_new0 (tensor_dim, 1);
+          g_assert (d != NULL);
```

Comment:
```g_new0()``` aborts the program on failure, which means this line looks unnecessary.

Reply:
+1